### PR TITLE
SPCTR-2918 Responsive CSS changes in popup, post layout, price list, section, separator, slider and child, social share and star rating

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-2918

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by rahulvarma722 — APPROVED** [review]

**Review by aapatre — CHANGES_REQUESTED** [review]
As per the JIRA card, the changes are due to the upcoming iFrame update in Gutenberg. Please add the same in the PR description since this is a significant fix.

cc: @AbhijitNage123

File: src/blocks/star-rating/styling.js [hunk 1]
```diff
@@ -6,9 +6,9 @@ import generateCSS from '@Controls/generateCSS';
 import generateCSSUnit from '@Controls/generateCSSUnit';
 import { getFallbackNumber } from '@Controls/getAttributeFallback';
 
-function styling( attributes, clientId, name ) {
+function styling( attributes, clientId, name, deviceType ) {
```
Comment (adi3890) [inline src/blocks/star-rating/styling.js:9]: in other Prs i see we are directly using block_id from attributes.
do we need clientId

Comment (AbhijitNage123) [reply src/blocks/star-rating/styling.js:9]: Yes its needed for block those uses clientId. some of the block are using block_id and some are using clientId.

Comment (adi3890) [reply src/blocks/star-rating/styling.js:9]: ok

**Review by adi3890 — APPROVED** [review]

**Pull request merged**